SessionMessagingKit/Sending & Receiving/Notifications/PushNotificationAPI.swift

```diff
@@ -435,7 +435,16 @@ public enum PushNotificationAPI {
         
         // If the metadata says that the message was too large then we should show the generic
         // notification (this is a valid case)
-        guard !notification.info.dataTooLong else { return (nil, notification.info, .successTooLong) }
+        guard !notification.info.dataTooLong else {
+            SNLog("Ignoring notification due to data being too long")
+            return (nil, notification.info, .successTooLong)
+        }
+        
+        // Only show notifcations for messages from default namespace
```

Comment:
No change needed for this (just a note for myself) - we will need to add the `groupMessages` namespace to this case as well

Reply:
Added a TODO for this